controls/datagrid/theming-and-styles/columns-styling.md

```diff
@@ -21,58 +21,88 @@ The styling mechanism is represented by the following properties:
 
->note By default, the indicator is represented by a string symbol that can be changed by using the `IndicatorText` and `IndicatorFontFamily` properties. For more details, refer to the [article on sorting the DataGrid]({%slug datagrid-sorting-overview%}).
+    You can check how the individual indicators look on Windows below:
 
-### Indicator Styling
+    ![Telerik DataGrid Filtering and Sorting indicators](../images/datagrid-styling-indicators-individual.png)
 
-Style the `Indicator` using the following properties:
+    * `Universal`&mdash;A single indicator will be used when any data operation (filtering, sorting or grouping) is applied. In case sorting is applied, the sort indicator won't be displayed, only the universal indicator. In case filtering is applied, in addition to the universal indicator, the filter indicator takes its active color (the active color can be modified through the `DataGridColumnHeaderStyle`'s `FilterIndicatorActiveTextColor` property).
 
-* `IndicatorColor`&mdash;Defines the color for the indicator part of the `ColumnHeader`.
-* `IndicatorText`&mdash;Defines the text for the indicator part of the `ColumnHeader`.  
-* `IndicatorMargin`&mdash;Defines the margin of the option indicator part of the `ColumnHeader`.
-* `Font Options`(`IndicatorFontAttributes`, `IndicatorFontFamily`, `IndicatorFontSize`)&mdash;Define the font options to the option indicator text. 
+    Check below how the indicators are displayed in `DataOperationIndicationMode`.`Universal`:
 
+    ![Telerik DataGrid Universal Indicator](../images/datagrid-styling-indicators-universal.png)
 
-### FilterIndicator Styling
+### Filter Indicator Styling
 
-Style the `FilterIndicator` using the following properties:
+The `FilterIndicator` enables users to show the Filtering UI and filter the data. Style the `FilterIndicator` using the following properties:
 
 * `FilterIndicatorTextColor`&mdash;Defines the color for the filter indicator part of the `ColumnHeader`.  
-* `FilterIndicatorText`&mdash;Defines the text for the filter indicator part of the `ColumnHeader`. For the `FilterIndicatorText` to appear the `FilterIndicatorFontFamily` should be specified.
+* `FilterIndicatorActiveTextColor`&mdash;Defines the color of the filter symbol when filtering is applied.
+* `FilterIndicatorText`&mdash;Defines the text for the filter indicator part of the `ColumnHeader`. For the `FilterIndicatorText` to appear, set the `FilterIndicatorFontFamily` property.
 * `FilterIndicatorMargin`&mdash;Defines the margin of the filter indicator part of the `ColumnHeader`.
 * `Font Options`(`FilterIndicatorFontAttributes`, `FilterIndicatorFontFamily`, `FilterIndicatorFontSize`)&mdash;Define the font options to the filter indicator text. 
 
+>note By default, the filter indicator is represented by a string symbol that can be changed by using the `FilterIndicatorText` and `FilterIndicatorFontFamily` properties. 
+
+### Sort Indicators Styling
```

Comment:
Is there a reason for using plural here? If not, consider switching to singular for consistency.

```suggestion
### Sort Indicator Styling
```